.gitignore

```diff
@@ -20,4 +17,4 @@ __pycache__
 docs/build
 
 # DB config gfile
-dbDetails.yml
\ No newline at end of file
+dbDetails.yml
```

Comment:
I assume dbDetails is still in the gitignore and all you added is a new line, right?

Reply:
No - I removed `config.yml` from .gitignore. I made a mistake in naming this branch, sorry for the confusion!